devel/libcudd/Portfile

```
@@ -3,10 +3,10 @@
 PortSystem          1.0
 
 name                libcudd
-version             2.5.0
+version             3.0.0
 categories          devel
 platforms           darwin
-maintainers         caltech.edu:SCL openmaintainer
+maintainers         cds.caltech.edu:slivingston openmaintainer
```

You should include your Github handle:

```suggestion
maintainers         {cds.caltech.edu:slivingston @slivingston} \
                    openmaintainer
```